### DISCUSSION TITLE
why can‘t I use it

### DISCUSSION THREAD

**Discussion by ADY-ZAT** [iptv-org/community#223] [Q&A] [answered]
![N)P70UG9PR)Q7 V %69ERBW](https://user-images.githubusercontent.com/87921949/126892802-14c8ea9d-f91f-4056-a540-a0f9a9bd05e7.png)
My Potplayer Can not play the link and show that the address wrong 

What is it mean?

**Comment by Sphinxroot** [comment]
try this please  https://raw.githubusercontent.com/iptv-org/iptv/master/channels/cn.m3u

**Reply by ADY-ZAT** [reply]
It can work

**Reply by ADY-ZAT** [reply]
and If I wanna Japan ?

**Comment by Sphinxroot** [comment]
select the one you want .. then click raw   .. copy and paste in potplayer  should be good :)
https://github.com/iptv-org/iptv/tree/master/channels  

https://raw.githubusercontent.com/iptv-org/iptv/master/channels/jp.m3u

or if you want them  all   use vlc , open a network stream , add https://raw.githubusercontent.com/iptv-org/iptv/master/index.m3u

**Reply by ADY-ZAT** [reply]
I know,but I can not use that link It show error

**Reply by Sphinxroot** [reply]
ive tested im potplayer  and some stream   wont start  but the use of  unique list will be show  https://i.imgur.com/shQlHR3.png

vlc whit the index will work  but not in potplayer 

this is vlc + https://raw.githubusercontent.com/iptv-org/iptv/master/index.m3u
https://i.imgur.com/SQ5hRn5.png

**Comment by Dum4G** [comment, marked as answer]
Link you have is a fancy redirect to https://raw.githubusercontent.com/iptv-org/iptv/gh-pages/countries/cn.m3u

Looks like your player doesn't support redirection links
